common/models.py

```diff
@@ -395,19 +395,19 @@ class CustomPage(Page, index.Indexed):
     def serve(self, request):
         return render(request, self.template, {
             'page': self,
-            'people': Person.objects.all(),
+            'people': Person.objects.all().prefetch_related('photo'),
```

Comment:
Drive-by: I think you can use `select_related` rather than `prefetch_related` since `photo` is a many-to-one relationship. That will save even more queries.